### PR TITLE
Added support for instance_warmup

### PR DESCRIPTION
**Pull request by jvasallo**
- Delay between new instance coming up and metrics feeding into ASG
Aggregate metrics

### CODE REVIEW

**Comment by jvasallo** [comment]
Need to review minutes as well.

File: _docs/configuration_files/application_json.rst [hunk 1]
```diff
@@ -260,6 +260,12 @@ was met
       - ``"Minimum"``
       - ``"Sum"``
 
+``instance_warmup`` : Time period to wait before adding metrics to Auto Scaling group
+
+   | *Type*: int
+   | *Default*: 600
+   | *Units*: seconds
+
```
Comment (sijis) [inline _docs/configuration_files/application_json.rst]: You should probably also update the `scaling_policy` example.

File: _docs/configuration_files/application_json.rst [hunk 2]
```diff
@@ -235,7 +235,7 @@ scaling policy will be attached
 
 ``threshold`` : Metrics value limit for scaling up
 
-   | *Type*: number
+   | *Type*: int
```
Comment (sijis) [inline _docs/configuration_files/application_json.rst]: Thanks for updating this.

**Comment by sijis** [comment]
Is this still a WIP?

**Comment by jvasallo** [comment]
Guess not 😄

**Pull request merged**